django/contrib/admin/static/admin/img/README.txt

```diff
@@ -1,7 +1,18 @@
-All icons are taken from Font Awesome (https://fontawesome.com/) project.
-The Font Awesome font is licensed under the SIL OFL 1.1:
-- https://scripts.sil.org/OFL
+## Icons License
 
-SVG icons source: https://github.com/encharm/Font-Awesome-SVG-PNG
-Font-Awesome-SVG-PNG is licensed under the MIT license (see file license
-in current folder).
+All icons in this directory are provided by [Font Awesome Free](https://fontawesome.com), version 6.x.
+
+### License for Font Awesome Icons:
+- The icons are licensed under the [Creative Commons Attribution 4.0 International (CC BY 4.0)](https://creativecommons.org/licenses/by/4.0/) license.
+- This license allows you to use, modify, and distribute the icons, provided proper attribution is given.
+
+### Attribution:
+To comply with the CC BY 4.0 license, please include the following attribution in your project:
```

Comment:
The actual text still seems to be missing?

Reply:
Yes, I'm going to delete that part